src/app/cases/styles.tsx

```diff
@@ -1,6 +1,17 @@
 import styled from 'styled-components';
 import COLORS from '@/styles/colors';
 
+export const AuthButtons = styled.div`
```

Comment:
@davidqing6432 thoughts on making this smaller? i feel like they're lowk really big relative to the filters rn